EpiAware/src/latentmodels/autoregressive.jl

```diff
@@ -0,0 +1,53 @@
+struct AR <: AbstractLatentModel
```

Comment:
Its usually better to use concrete types for fields of structs. This looks like you should make it a [parameteric](https://docs.julialang.org/en/v1/manual/types/#Parametric-Types) typed struct.